### ISSUE TITLE
Create liftover failure event when you can't use ClinGen Allele registry

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#1103]
During new liftover work, I made a fake AlleleLiftover error/failure for historical alleles that would have failed ClinGen liftover

But during the normal process, we just don't perform a liftover we know will failure.

Thus it's not obvious why we didn't try ClinGen - we should create a failure entry if we can't even try

**Comment by davmlaw** [comment]
The original issue works

While Thomas was testing, he pointed out badly worded error message

So: need to come up with better wording, then edit, patch, deploy etc.

Or, if you are waiting on me - you could break up the rewording into a new issue (for Q3) and ship with the current message

**Comment by davmlaw** [comment]
Context: In the method `snpdb.liftover._liftover_using_dest_variant_coordinate`  we set the message to be `clingen_failure_message = "No ClinGenAllele for variant"` which isn't very informative

I think we only get in this situation if we SKIP clingen, as if we did go there and failed, we'd have stored an error, which would have been found via the `AlleleLiftover.has_existing_failure` so this is more accurately we skipped this variant (for whatever reason) - I think `snpdb.models.models_variant.Variant.can_have_clingen_allele` to be precise so either too long, or is symbolic and not del/dup/inv

Maybe we could alter the code around can_have_clingen_allele to actually tell us why?

**Comment by davmlaw** [comment]
@lhomas - deployed and ready for testing
